kong.conf.default

```diff
@@ -1447,6 +1447,34 @@
                                  # it might take more time to propagate changes
                                  # to each individual worker.
 
+#router_flavor = traditional_compatible
+                                 # Selects the router implementation to use when
+                                 # performing request routing. Incremental router
+                                 # rebuild is available when the flavor is set
+                                 # to either `expressions` or
+                                 # `traditional_compatible` which could
+                                 # significantly shorten rebuild time for large
+                                 # number of routes.
```

Comment:
The maintainers of the project have an understanding of what "incremental router rebuild" is but the operator does not and should not need to either. That is more detail than necessary.

I'd recommend rewording this as "recommend values are foo and bar for performance with a large number of routes".

Reply:
I feel hesitant to make such change. Even for shorter number of routes, the new router is still recommended as it also runs faster in these cases. We should not pitch the new router as just a performance improvement only because that is missing the main point of this change.

"which could significantly shorten rebuild time for large number of routes" should explain the term "incremental router rebuild" well. We even mentioned this term to some customers in the past and it does not seems to be very difficult to undersatand.